internal/receiver/discoveryreceiver/README.md

```diff
@@ -0,0 +1,334 @@
+Attributes:
+     -> type: STRING(endpoint)
+     -> labels: STRING(map[maintainer:NGINX Docker Maintainers <docker-maint@nginx.com>])
+     -> id: STRING(2567fdbc764706d29120b01efcc3a310d87e9e121ec9debbc977d66f5497cdda:80)
+     -> tag: STRING(latest)
+     -> port: STRING(80)
+     -> alternate_port: STRING(0)
+     -> command: STRING(nginx -g daemon off;)
+     -> container_id: STRING(2567fdbc764706d29120b01efcc3a310d87e9e121ec9debbc977d66f5497cdda)
+     -> transport: STRING(TCP)
+     -> name: STRING(ecstatic_davinci)
+     -> image: STRING(nginx)
+     -> host: STRING(172.17.0.2)
+     -> endpoint: STRING(172.17.0.2:80)
+Trace ID:
+Span ID:
+Flags: 0
+LogRecord #2
+ObservedTimestamp: 2022-07-27 19:00:32.305784658 +0000 UTC
+Timestamp: 2022-07-27 19:00:32.305784612 +0000 UTC
+Severity: info
+Body: {"alternate_port":0,"command":"redis-server","container_id":"5f7f5f007f798c59a60c765e566db5d22bff59c614268db8d1b9abbc3ee70bf7","endpoint":"172.17.0.3:6379","host":"172.17.0.3","id":"5f7f5f007f798c59a60c765e566db5d22bff59c614268db8d1b9abbc3ee70bf7:6379","image":"redis","labels":{},"name":"beautiful_clarke","port":6379,"tag":"latest","transport":"TCP","type":"container"}
+Attributes:
+     -> type: STRING(endpoint)
+     -> command: STRING(redis-server)
+     -> transport: STRING(TCP)
+     -> endpoint: STRING(172.17.0.3:6379)
+     -> port: STRING(6379)
+     -> image: STRING(redis)
+     -> tag: STRING(latest)
+     -> alternate_port: STRING(0)
+     -> container_id: STRING(5f7f5f007f798c59a60c765e566db5d22bff59c614268db8d1b9abbc3ee70bf7)
+     -> host: STRING(172.17.0.3)
+     -> labels: STRING(map[])
+     -> id: STRING(5f7f5f007f798c59a60c765e566db5d22bff59c614268db8d1b9abbc3ee70bf7:6379)
+     -> name: STRING(beautiful_clarke)
+Trace ID:
+Span ID:
+Flags: 0
+```
+
+## Example Usage
+
+The following Collector configuration will create a Discovery receiver instance that receives
+endpoints from a [Docker Observer](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/extension/observer/dockerobserver/README.md)
+that reports log records denoting the status of a [Smart Agent receiver](https://github.com/signalfx/splunk-otel-collector/blob/main/internal/receiver/smartagentreceiver/README.md)
+configured to use the `collectd/redis` monitor. The `status` mapping comprises entries that signal the Smart Agent
+receiver has been instantiated with a `successful`, `partial`, or `failed` status, based on reported `metrics` or
+recorded application log `statements`.
+
+The Redis receiver will warrant a status log record from the Discovery receiver of...
+
```

Comment:
The Redis receiver warrants the following a status log record from the Discovery receiver: